pkg/executor/executor.go

```diff
@@ -202,28 +201,29 @@ func (k *KubernetesExecutor) _launchTask(taskInfo *mesos.TaskInfo, podFullName s
 			continue // attempt to recover
 		}
 
+		running := mesos.TaskState_TASK_RUNNING
```

Comment:
this is awkward. need a mesosutil equivalent so that we're not creating temp variables just to get a pointer to a const